### ISSUE TITLE
Add config item for database storage location

### ISSUE THREAD

**Issue by j-frost** [Source-Explorers/tep#1]

**Comment by ghost** [comment]
I would propose to have a few system paths in which TEP looks for its configuration and the ability to override the used configuration path with a command-line switch.

The paths I would code into TEP should be at least and in that order:

- the current directory (`./`)
- the default config dir in $HOME (`$HOME/.config/tep/`)
- old style config directory (`$HOME/.tep`)
- system-wide config file (`/etc/tep`)

For the command-line switch, the simplest would be to use the [internal argument](https://erlang.org/doc/man/erl.html) list:
```
% erl -myflag 1
1> init:get_argument(myflag).
{ok,[["1"]]}
2> init:get_plain_arguments().
[]
```

For the time being, I would prefer this simple approach to using a full blown library.

What is your opinion on this?

**Comment by j-frost** [comment]
I agree with your approach @ndoering. Does this warrant its own ticket?

**Comment by ghost** [comment]
@j-frost : Indeed, that might be the best way. I misunderstood the task and thought it would be to implement the configuration file in itself.
